GW2EIEvtcParser/EIData/Statistics/Point3D.cs

```diff
@@ -117,6 +120,31 @@ public Point3D(Point3D a, Point3D b, float ratio)
             Z = Mix(a.Z, b.Z, ratio);
         }
 
+        /// <summary>
+        /// Calculate the facing <see cref="Point3D"/> during a <see cref="AbstractCastEvent"/>.
+        /// </summary>
+        /// <param name="replay">Combat Replay.</param>
+        /// <param name="c">Cast event.</param>
+        /// <param name="castTimeDuration">Duration of the cast time.</param>
+        /// <returns>Accurate facing point during the cast duration.</returns>
+        public static Point3D GetFacingPoint3D(CombatReplay replay, AbstractCastEvent c, int castTimeDuration)
```

Comment:
Could be a non static method in CombatReplay.cs no?

Reply:
Or maybe even in AbstractSingleActor where an API for position/rotation fetching already exists.

Reply:
Don't have any strong opinion about this one, I just needed it outside of Skorvald's class and since it returns a Point3D I've located it there. Can leave it here for now and you can relocate it in the cleanup branch later on?

Reply:
Sure!